Documentation/subcommands/prepare.md

```diff
@@ -0,0 +1,32 @@
+# rkt prepare
```

Comment:
Could you add note explaining that the arguments should all be the same between prepare/run, and that e.g. `rkt prepare + rkt run-prepared` is semantically equivalent to `rkt run`?